src/main/resources/application.properties

```diff
@@ -127,3 +127,4 @@ azure.type.name=azure
 #Redis Cache Service properties
 redis.host.name=redisHostName
 redis.port=6379
+redis.timeout=redisTimeOutNeeded
```

Comment:
Provide 84600 seconds (i.e. one day) as default value... 
add an comment above and mention that timeout value is in seconds...